specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-03-01-preview/EntityQueries.json

```diff
@@ -0,0 +1,651 @@
+          "description": "List of the desired output types to be constructed from the result",
+          "items": {
+            "$ref": "#/definitions/EntityInnerType",
+            "description": "output entity type"
+          },
+          "type": "array"
+        },
+        "queryTemplate": {
+          "description": "The template query string to be parsed and formatted",
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "ActivityEntityQuery": {
+      "description": "Represents Activity entity query.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/EntityQuery"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ActivityEntityQueriesProperties",
+          "description": "Activity entity query properties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object",
+      "x-ms-discriminator-value": "Activity"
+    },
+    "ActivityCustomEntityQuery": {
+      "description": "Represents Activity entity query.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/CustomEntityQuery"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ActivityEntityQueriesProperties",
+          "description": "Activity entity query properties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object",
+      "x-ms-discriminator-value": "Activity"
+    },
+    "ActivityEntityQueriesProperties": {
+      "description": "Describes activity entity query properties",
+      "properties": {
+        "title": {
+          "description": "The entity query title",
+          "type": "string"
+        },
+        "content": {
+          "description": "The entity query content to display in timeline",
+          "type": "string"
+        },
+        "description": {
+          "description": "The entity query description",
+          "type": "string"
+        },
+        "queryDefinitions": {
+          "description": "The Activity query definitions",
+          "properties": {
+            "query": {
+              "description": "The Activity query to run on a given entity",
+              "type": "string"
+            }
+          },
+          "type": "object"
+        },
+        "inputEntityType": {
+          "$ref": "#/definitions/EntityInnerType",
+          "description": "The type of the query's source entity"
+        },
+        "requiredInputFieldsSets": {
+          "description": "List of the fields of the source entity that are required to run the query",
+          "items": {
+            "description": "Sub sets of the field of the source entity that are required to run the query",
+            "items": {
+              "description": "Required input field name",
+              "type": "string"
+            },
+            "type": "array"
+          },
+          "type": "array"
+        },
+        "entitiesFilter": {
```

Comment:
why is this an object type?  What is an example of this property?

Reply:
same example: "examples/entityQueries/CreateEntityQueryActivity.json"
this is being translate to a dictionary:
{ 
  "key1": ["value1", "value2"]
}

Reply:
Although open ended models like this are discouraged swagger does support dictionaries and this should be modeled more tightly.  See "additionalProperties" in https://swagger.io/specification/v2/

Reply:
Done